client/src/components/Review/ReviewList/ReviewListItem/ReviewDetail/ReviewDetail.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Mock } from '../../../../../containers/ReviewContainer';
+import ReviewDetailImage from './ReviewDetailImage/ReviewDetailImage';
+
+type Props = {
+  review: Mock;
+};
+
+const REVIEW_DETAIL_WIDTH = 950;
+const IMAGE_GRID_COLUMNS = 5;
+
+const Container = styled.div`
+  width: ${REVIEW_DETAIL_WIDTH}px;
+  margin: 20px auto 0;
+  padding: 20px;
+  border: 1px solid ${(props) => props.theme.color.grey1};
+  border-radius: 5px;
+`;
+
+const ReviewImageList = styled.ul`
+  width: 100%;
+  margin-bottom: 20px;
+  display: grid;
+  grid-template-columns: repeat(${IMAGE_GRID_COLUMNS}, 1fr);
+  grid-gap: 10px;
+`;
+
+const ReviewContent = styled.p`
+  line-height: 1.6;
+  font-size: ${(props) => props.theme.fontSize.small};
+`;
+
+const ReviewDetail = (props: Props): JSX.Element => {
+  const { review } = props;
+  const reviewImages = review.reviewImages.map((imageUrl, i) => (
+    <ReviewDetailImage
+      key={i}
+      imageUrl={imageUrl}
+      height={REVIEW_DETAIL_WIDTH / IMAGE_GRID_COLUMNS}
+    />
+  ));
+  const shouldRenderReviewImageList = reviewImages.length > 0;
```

Comment:
`useMemo`를 사용하는건 어떨까요?
(아래 코드가 정확한진 모르겠네요 ㅎㅎ)
```suggestion
const shouldRenderReviewImageList = useMemo(() => reviewImages.length > 0, [reviewImages.length])
```

Reply:
진우님 코멘트를 보고 `useMemo`를 조금 찾아봤는데 여기서는 굳이 사용할 필요가 없을 것 같다는 생각이 들었습니다.

우선은 연산이 복잡하지 않고, ReviewDetail 컴포넌트는 자체적으로 상태를 가지고 있지 않아서 어차피 리랜더링이 발생하는 경우는 props인 review가 변경되는 경우밖에 없을 것 같습니다. (그것도 실제로는 일어나지 않겠지만).

아무튼 그렇게 되면 `useMemo`의 사용 여부와 관계없이 `shouldRenderReviewImageList`는 다시 연산을 하기 때문에 크게 의미가 없을 듯합니다. 제가 혹시 뭔가 놓치고 있는게 있다면 알려주세요 🙂